Fix Excalidraw

Excalidraw ne fonctionne plus depuis le passage à `pnpm`

```bash
documentation_1      | yarn run v1.22.19
documentation_1      | warning Skipping preferred cache folder "/.cache/yarn" because it is not writable.
documentation_1      | warning Selected the next writable cache folder in the list, will be "/tmp/.yarn-cache-1001".
documentation_1      | $ react-scripts start
documentation_1      | warning Cannot find a suitable global folder. Tried these: "/usr/local, /.yarn"
documentation_1      | WARN 2022/10/21 09:21:20 found no layout file for "JSON" for layout "search" for kind "section": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.



documentation_1      | ./src/components/LibraryMenuItems.tsx
documentation_1      | Module not found: Can't resolve 'lodash' in '/excalidraw/src/components'
documentation_1      | Compiling...
documentation_1      | Failed to compile.
documentation_1      | 
documentation_1      | ./src/components/LibraryMenuItems.tsx
documentation_1      | Module not found: Can't resolve 'lodash' in '/excalidraw/src/components'
documentation_1      | Compiling...
documentation_1      | Failed to compile.
documentation_1      | 
documentation_1      | ./src/components/LibraryMenuItems.tsx
documentation_1      | Module not found: Can't resolve 'lodash' in '/excalidraw/src/components'
```

--- a/.dockerignore
+++ b/.dockerignore
@@ -1,1 +1,3 @@
 hugo/config.toml
+hugo/.hugo_build.lock
+hugo/content/*

--- a/.gitignore
+++ b/.gitignore
@@ -3,3 +3,4 @@ hugo/config.toml
 hugo/resources/_gen/*
 hugo/public/*
 hugo/content
+hugo/archetypes

--- a/Dockerfile
+++ b/Dockerfile
@@ -2,7 +2,7 @@ FROM node:16-slim AS base
 
 RUN apt-get update && apt-get install -y \
     wget \
-    git && rm -rf /var/lib/apt/lists/* && wget https://github.com/gohugoio/hugo/releases/download/v0.104.3/hugo_extended_0.104.3_linux-amd64.deb && dpkg -i hugo_extended_0.104.3_linux-amd64.deb && npm install -g pnpm
+    git && rm -rf /var/lib/apt/lists/* && wget https://github.com/gohugoio/hugo/releases/download/v0.104.3/hugo_extended_0.104.3_linux-amd64.deb && dpkg -i hugo_extended_0.104.3_linux-amd64.deb
 
 FROM base AS hugo
 
@@ -12,11 +12,12 @@ RUN chmod -R 777 /documentation && rm -f /documentation/.hugo_build.lock
 FROM base AS excalidraw
 
 RUN git clone -b v0.12.0 https://github.com/excalidraw/excalidraw.git excalidraw
-RUN cd excalidraw && pnpm install && npx browserslist@latest --update-db
+RUN cd excalidraw && yarn && npx browserslist@latest --update-db
 RUN mkdir -p /excalidraw/node_modules/.cache && chmod -R 777 /excalidraw/node_modules/.cache
 
 FROM node:16-slim AS production
 
+RUN mkdir -p /.yarn && touch /.yarnrc && mkdir -p /.cache && chmod -R 777 /.yarn /.yarnrc /.cache
 COPY --from=hugo /documentation /documentation
 COPY --from=hugo /usr/local/bin/hugo /usr/local/bin/hugo
 COPY --from=excalidraw /excalidraw /excalidraw

--- a/Readme.md
+++ b/Readme.md
@@ -170,7 +170,7 @@ summaryLength = 30
     meta_twitter_creator = "@zerostaticio"
 ```
 
-Il suffit alors de monter se fichier dans le conteneur Docker, par exemple avec Docker Compose :
+Il suffit alors de monter se fichier dans le conteneur Docker, par exemple avec Docker Compose ou l'on change aussi le port de Hugo en `6666` et le port exposé de Excalidraw en `6667`:
 
 ```yaml
 version: "3.7"
@@ -181,9 +181,11 @@ services:
     volumes:
       - ./documentations:/documentation/content
       - ./hugo-config.toml:/documentation/config.toml
+    environment:
+      - HUGO_PORT=6666
     ports:
-      - 3000:3000
-      - 1313:1313
+      - 6666:6666
+      - 6667:3000
 ```
 
 ## Publication

--- a/start-documentation.sh
+++ b/start-documentation.sh
@@ -1,17 +1,9 @@
 #!/bin/bash
-
+PORT="${HUGO_PORT:=1313}"
 # Start Excalidraw
 cd /excalidraw && yarn start &
 
 # Setup hugo configuration if not exist (aka nod mounted in the docker container)
 cp -n /documentation/config-default.toml /documentation/config.toml
 # Start hugo
-cd /documentation && hugo server --bind 0.0.0.0 --navigateToChanged
-
-# &
-
-# # Wait for any process to exit
-# wait -n
-
-# # Exit with status of process that exited first
-# exit $?
+cd /documentation && hugo server --port $PORT --bind 0.0.0.0 --navigateToChanged

